3.Linear_Modeling/1b.fit_linear_model_2b.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        # treatment_df = feature_df[["Metadata_inducer1", "Metadata_inducer1_concentration"]]
</pre><p>consider removing this if you're no longer using it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>